meta/main.yml

```diff
@@ -29,12 +29,14 @@ galaxy_info:
         - bookworm
     - name: Fedora
       versions:
-        - 34
-        - 35
+        - "35"
+        - "36"
+        - "37"
     - name: Ubuntu
       versions:
         - bionic
         - focal
+        - jammy
   role_name: openvpn
 
 dependencies:
```

Comment:
Please explicitly add [cisagov/ansible-role-python](https://github.com/cisagov/ansible-role-python) as a dependency. When `pip` is installed as a package in [cisagov/ansible-role-pip](https://github.com/cisagov/ansible-role-pip) it will install whatever `python` packages it has as dependencies. We should ensure a consistent Python presence by using our role that installs Python and its dependencies if we need Python.

Reply:
Please see commit d147f24.